extra-dev/packages/coq-metacoq-erasure/coq-metacoq-erasure.dev/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://metacoq.github.io/metacoq"
+dev-repo: "git+https://github.com/MetaCoq/metacoq.git#coq-8.11"
```

Comment:
Bad copy-paste?

Reply:
that's what it says in the opam file in the repo

Reply:
Regardless of what it says in the repo opam file, please change to:
```
git+https://github.com/MetaCoq/metacoq.git#master
```